pkg/webreg/webreg.go

```diff
@@ -706,6 +706,112 @@ func WebRegHandler(regAgent agents.RegistryAgent, confAgent agents.ConfigAgent)
 	}
 }
 
+// multiLineHighlightJS allows to append -$NUMBER to one of the line anchors
+// (e.G. `#line18-20`) and will then highlight all the lines.
+const multiLineHighlightJS = `<script>
+function getLinesToHighlight() {
+  let result = [];
+  url = window.location.href;
+  anchorSplitUrl = url.split("#");
+  // No results
+  if (anchorSplitUrl.length < 2) {
+    return result
+  };
```

Comment:
No need for a semicolon here, but one after the return statement above would be nice.

Reply:
done